### ISSUE TITLE
how to implement

### ISSUE THREAD

**Issue by yshmarov** [colinwahl/react-recurring-select#1]
can you please give documentation about implementation into a rails app?...

**Comment by jenyeeiam** [comment]
Any progress on this? I just cloned and it works, now I want to know how to implement with my current app

**Comment by donatoaz** [comment]
I don't think this module is still being maintained.... has anyone figured out how to build it? I've made changes to it on a fork, but it does not have a `dist` script and the `transform` that it does have, does not work...
